core/src/main/java/tech/pegasys/web3signer/core/Eth1Runner.java

```diff
@@ -74,23 +69,18 @@ public Eth1Runner(final BaseConfig baseConfig, final Eth1Config eth1Config) {
   }
 
   @Override
-  protected String getOpenApiSpecResource() {
-    return "eth1/web3signer.yaml";
-  }
-
-  @Override
-  protected Router populateRouter(final Context context) {
-    final RouterBuilder routerBuilder = context.getRouterBuilder();
+  protected void populateRouter(final Context context) {
+    final Router router = context.getRouter();
     final LogErrorHandler errorHandler = context.getErrorHandler();
     final ArtifactSignerProvider signerProvider = context.getArtifactSignerProvider();
 
     addPublicKeysListHandler(
-        routerBuilder, signerProvider, ETH1_LIST.name(), context.getErrorHandler());
+        router, signerProvider, "/api/v1/eth1/publicKeys", context.getErrorHandler());
```

Comment:
it might be a good idea to create an interface, may be `Eth1Routes`, `Eth2Routes`, that list all the constants for the route paths.

Reply:
extracted some constants to the runners